Added docs for enabling Sippy with Wrangler, GCS provider (#12652)
cloudflare/cloudflare-docs: content/r2/data-migration/sippy.md

## Get started with Sippy

> Before getting started, you'll need:

Reviewer: ```suggestion
Before getting started, you will need:
```

- An existing R2 bucket. If you don’t already have one, refer to [Create buckets](/r2/buckets/create-buckets/).
- [API credentials](/r2/data-migration/sippy/#create-credentials-for-storage-providers) for your source object storage bucket.
- Cloudflare R2 Access Key ID and Secret Access Key with read and write permissions. For more information, refer to [Authentication](/r2/api/s3/tokens/).

### Enable Sippy via Wrangler

#### Set up Wrangler

To begin, install [`npm`](https://docs.npmjs.com/getting-started). Then [install Wrangler, the Developer Platform CLI](/workers/wrangler/install-and-update/).

#### Enable Sippy on your R2 bucket

Log in to Wrangler with the [`wrangler login` command](/workers/wrangler/commands/#login). Then run the [`r2 bucket sippy enable` command](/workers/wrangler/commands/#sippy-enable):

```sh
$ npx wrangler r2 bucket sippy enable <BUCKET_NAME>
```

This will prompt you to select between supported object storage providers and lead you through setup.

> ### Enable Sippy via API
> The example below shows how to enable Sippy for an R2 bucket with the AWS S3 provider using cURL. For information about getting started with the Cloudflare API, refer to [Make API calls](/fundamentals/api/how-to/make-api-calls/).

Reviewer: ```suggestion
### Enable Sippy via API

The following example shows how to enable Sippy for an R2 bucket with the AWS S3 provider using cURL. For information about getting started with the Cloudflare API, refer to [Make API calls](/fundamentals/api/how-to/make-api-calls/).
```

## Supported cloud storage providers

> We currently support copying data from the following cloud object storage providers to R2:

Reviewer: ```suggestion
Cloudflare currently supports copying data from the following cloud object storage providers to R2:
```

Actions not listed above have no change in behavior. For more information, refer to [Workers API reference](/r2/api/workers/workers-api-reference/) or [S3 API compatibility](/r2/api/s3/api/).

## Create credentials for storage providers

### Amazon S3

> To copy objects from Amazon S3, Sippy requires access permissions to your bucket. While you can use any AWS Identity and Access Management (IAM) user credentials with the correct permissions, we recommend you create a user with a narrow set of permissions.

Reviewer: ```suggestion
To copy objects from Amazon S3, Sippy requires access permissions to your bucket. While you can use any AWS Identity and Access Management (IAM) user credentials with the correct permissions, Cloudflare recommends you create a user with a narrow set of permissions.
```

You can now use both the Access Key ID and Secret Access Key when enabling Sippy.

> ### Google Cloud Storage
> To copy objects from Google Cloud Storage (GCS), Sippy requires access permissions to your bucket. We recommend using the Google Cloud predefined `Storage Object Viewer` role.

Reviewer: ```suggestion
### Google Cloud Storage

To copy objects from Google Cloud Storage (GCS), Sippy requires access permissions to your bucket. Cloudflare recommends using the Google Cloud predefined `Storage Object Viewer` role.
```

To create credentials with the correct permissions:
1. Log in to your Google Cloud console.

> 2. Navigate to **IAM & Admin** > **Service Accounts**.

Reviewer: ```suggestion
2. Go to **IAM & Admin** > **Service Accounts**.
```

3. Create a service account with the predefined `Storage Object Viewer` role.

> 4. Navigate to the **Keys** tab of the service account you created.

Reviewer: ```suggestion
4. Go to the **Keys** tab of the service account you created.
```
